benchmarks/benchmarks.py

```diff
@@ -0,0 +1,101 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+from arcticdb import Arctic
+from arcticdb.version_store.library import (
+    WritePayload,
+    ReadRequest
+)
+
+import pandas as pd
+import numpy as np
+
+def generate_pseudo_random_dataframe(n, freq="S", end_timestamp="1/1/2023"):
+    """
+    Generates a Data Frame with 2 columns (timestamp and value) and N rows
+    - timestamp contains timestamps with a given frequency that end at end_timestamp
+    - value contains random floats that sum up to approximately N, for easier testing/verifying
```

Comment:
We could write deterministic tests by seeding `numpy.random` with a raodom seed and recording the seed as metadata with the dataframe. Then the verification step could reconstruct the expected result from the seed in the metadata.

Reply:
yeah that makes more sense